{"uid":"c4134256ef1e34c7","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1703811730268,"stop":1703811740513,"duration":10245},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e033bd8d33 <unknown>\n#1 0x55e033895f87 <unknown>\n#2 0x55e0338e5c0e <unknown>\n#3 0x55e0338e5d11 <unknown>\n#4 0x55e033931004 <unknown>\n#5 0x55e03390f36d <unknown>\n#6 0x55e03392e297 <unknown>\n#7 0x55e03390f0e3 <unknown>\n#8 0x55e0338d7044 <unknown>\n#9 0x55e0338d844e <unknown>\n#10 0x55e033b9d861 <unknown>\n#11 0x55e033ba1785 <unknown>\n#12 0x55e033b8b285 <unknown>\n#13 0x55e033ba241f <unknown>\n#14 0x55e033b6f20f <unknown>\n#15 0x55e033bc6028 <unknown>\n#16 0x55e033bc61f7 <unknown>\n#17 0x55e033bd7ed4 <unknown>\n#18 0x7fbd88494ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f3790661510>\nemail = 'elias@gmail.com', password = '12345Tr'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"03a2e71bce5d0c21e549a491c5fa3041\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f379071df80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55e033bd8d33 <unknown>\nE       #1 0x55e033895f87 <unknown>\nE       #2 0x55e0338e5c0e <unknown>\nE       #3 0x55e0338e5d11 <unknown>\nE       #4 0x55e033931004 <unknown>\nE       #5 0x55e03390f36d <unknown>\nE       #6 0x55e03392e297 <unknown>\nE       #7 0x55e03390f0e3 <unknown>\nE       #8 0x55e0338d7044 <unknown>\nE       #9 0x55e0338d844e <unknown>\nE       #10 0x55e033b9d861 <unknown>\nE       #11 0x55e033ba1785 <unknown>\nE       #12 0x55e033b8b285 <unknown>\nE       #13 0x55e033ba241f <unknown>\nE       #14 0x55e033b6f20f <unknown>\nE       #15 0x55e033bc6028 <unknown>\nE       #16 0x55e033bc61f7 <unknown>\nE       #17 0x55e033bd7ed4 <unknown>\nE       #18 0x7fbd88494ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1703811689332,"stop":1703811689332,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1703811689332,"stop":1703811689332,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1703811689333,"stop":1703811689333,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e033bd8d33 <unknown>\n#1 0x55e033895f87 <unknown>\n#2 0x55e0338e5c0e <unknown>\n#3 0x55e0338e5d11 <unknown>\n#4 0x55e033931004 <unknown>\n#5 0x55e03390f36d <unknown>\n#6 0x55e03392e297 <unknown>\n#7 0x55e03390f0e3 <unknown>\n#8 0x55e0338d7044 <unknown>\n#9 0x55e0338d844e <unknown>\n#10 0x55e033b9d861 <unknown>\n#11 0x55e033ba1785 <unknown>\n#12 0x55e033b8b285 <unknown>\n#13 0x55e033ba241f <unknown>\n#14 0x55e033b6f20f <unknown>\n#15 0x55e033bc6028 <unknown>\n#16 0x55e033bc61f7 <unknown>\n#17 0x55e033bd7ed4 <unknown>\n#18 0x7fbd88494ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f3790661510>\nemail = 'elias@gmail.com', password = '12345Tr'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"03a2e71bce5d0c21e549a491c5fa3041\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f379071df80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55e033bd8d33 <unknown>\nE       #1 0x55e033895f87 <unknown>\nE       #2 0x55e0338e5c0e <unknown>\nE       #3 0x55e0338e5d11 <unknown>\nE       #4 0x55e033931004 <unknown>\nE       #5 0x55e03390f36d <unknown>\nE       #6 0x55e03392e297 <unknown>\nE       #7 0x55e03390f0e3 <unknown>\nE       #8 0x55e0338d7044 <unknown>\nE       #9 0x55e0338d844e <unknown>\nE       #10 0x55e033b9d861 <unknown>\nE       #11 0x55e033ba1785 <unknown>\nE       #12 0x55e033b8b285 <unknown>\nE       #13 0x55e033ba241f <unknown>\nE       #14 0x55e033b6f20f <unknown>\nE       #15 0x55e033bc6028 <unknown>\nE       #16 0x55e033bc61f7 <unknown>\nE       #17 0x55e033bd7ed4 <unknown>\nE       #18 0x7fbd88494ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1703811730268,"stop":1703811740513,"duration":10245},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e033bd8d33 <unknown>\n#1 0x55e033895f87 <unknown>\n#2 0x55e0338e5c0e <unknown>\n#3 0x55e0338e5d11 <unknown>\n#4 0x55e033931004 <unknown>\n#5 0x55e03390f36d <unknown>\n#6 0x55e03392e297 <unknown>\n#7 0x55e03390f0e3 <unknown>\n#8 0x55e0338d7044 <unknown>\n#9 0x55e0338d844e <unknown>\n#10 0x55e033b9d861 <unknown>\n#11 0x55e033ba1785 <unknown>\n#12 0x55e033b8b285 <unknown>\n#13 0x55e033ba241f <unknown>\n#14 0x55e033b6f20f <unknown>\n#15 0x55e033bc6028 <unknown>\n#16 0x55e033bc61f7 <unknown>\n#17 0x55e033bd7ed4 <unknown>\n#18 0x7fbd88494ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"bb86a35428a11009","name":"public ip address","source":"bb86a35428a11009.txt","type":"text/plain","size":12},{"uid":"2045a120954957ed","name":"Cookies","source":"2045a120954957ed.json","type":"application/json","size":4404},{"uid":"977ebb8e9652e80e","name":"Session Storage","source":"977ebb8e9652e80e.json","type":"application/json","size":358},{"uid":"6acaa0ef4933f2b2","name":"Local Storage","source":"6acaa0ef4933f2b2.json","type":"application/json","size":208},{"uid":"4a0c3b67ea040f66","name":"Console Logs","source":"4a0c3b67ea040f66.json","type":"application/json","size":2},{"uid":"97ac2b55fab0da6c","name":"Full Page Screenshot","source":"97ac2b55fab0da6c.png","type":"image/png","size":1283197},{"uid":"7cabb5dfead8d998","name":"URL","source":"7cabb5dfead8d998.uri","type":"text/uri-list","size":23},{"uid":"fe4ce839a1359390","name":"Network Logs","source":"fe4ce839a1359390.json","type":"application/json","size":22922}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az525-405"},{"name":"thread","value":"2255-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"c4134256ef1e34c7.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}